Lib/typing.py

```diff
@@ -2753,7 +2753,16 @@ def __new__(cls, typename, bases, ns):
         return nm_tpl
 
 
-def NamedTuple(typename, fields=None, /, **kwargs):
+class _Sentinel:
+    __slots__ = ()
+    def __repr__(self):
+        return '<sentinel>'
+
+
+_sentinel = _Sentinel()
```

Comment:
I think that `= object()` is good enough for this case.

Reply:
I disagree. Using `= object()` would mean that the displayed signature would be very ugly if somebody did `help(typing.NamedTuple)` in the REPL

Reply:
Also, we'll be able to reuse the `_Sentinel` class when we do #105570 :)